### ISSUE TITLE
Missing example images for transform `SavitzkyGolaySmooth` and `RandRicianNoise`

### ISSUE THREAD

**Issue by Nic-Ma** [Project-MONAI/MONAI#3491]
**Is your feature request related to a problem? Please describe.**
I checked the document of intensity transforms, found that we didn't add example images for transform `SavitzkyGolaySmooth` and `RandRicianNoise`:
https://github.com/Project-MONAI/MONAI/blob/dev/docs/source/transforms.rst#RandRicianNoise
https://github.com/Project-MONAI/MONAI/blob/dev/docs/source/transforms.rst#SavitzkyGolaySmooth

Would be nice to add them.

**Comment by Nic-Ma** [comment]
And BTW, when I ran the program to generate images, I faced an error:
```
root@apt-sh-ai:/workspace/data/medical/MONAI# python monai/transforms/utils_create_transform_ims.py 
2021-12-15 08:55:59,318 - INFO - Expected md5 is None, skip md5 check for file /workspace/data/medical/MarsAtlas-MNI-Colin27.zip.
2021-12-15 08:55:59,318 - INFO - File exists: /workspace/data/medical/MarsAtlas-MNI-Colin27.zip, skipped downloading.
2021-12-15 08:55:59,318 - INFO - Writing into directory: /workspace/data/medical/MarsAtlas-MNI-Colin27.
Traceback (most recent call last):
  File "monai/transforms/utils_create_transform_ims.py", line 647, in <module>
    create_transform_im(AsDiscrete, dict(to_onehot=2, threshold=10), data, is_post=True, colorbar=True)
  File "monai/transforms/utils_create_transform_ims.py", line 418, in create_transform_im
    data_tr = transform(deepcopy(data_in))
  File "/workspace/data/medical/MONAI/monai/transforms/post/array.py", line 244, in __call__
    img_t = one_hot(img_t, num_classes=to_onehot, dim=0)
  File "/workspace/data/medical/MONAI/monai/networks/utils.py", line 91, in one_hot
    labels = o.scatter_(dim=dim, index=labels.long(), value=1)
RuntimeError: index 116 is out of bounds for dimension 0 with size 2
```

Thanks.

**Comment by Nic-Ma** [comment]
Hi @wyli @rijobro ,

Can you run `utils_create_transform_ims.py` in the latest codebase?
I am fixing the missing images and found the issue.

Thanks.

**Comment by Nic-Ma** [comment]
I found the root cause of AsDiscrete issue, seems the source image of LABEL changed to (0 - 255), fixed in PR #3494 .

Thanks.
